tools/ImportTool.Shared/RequestGenerator.cs

```diff
@@ -0,0 +1,111 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+using Task = System.Threading.Tasks.Task;
+
+namespace ImportTool
+{
+    public static class RequestGenerator
+    {
+        public static async Task<string> GenerateImportRequest(string connectionString, string prefix, int maxFileNumber)
```

Comment:
we should let customer provide account & key

Reply:
Fixed